db-server/test/backend/remote.js

```diff
@@ -1524,8 +1524,8 @@ module.exports = function(cfg, makeServer) {
             .spread((sessionTokenResp, keyFetchTokenResp) => {
               respOk(sessionTokenResp)
               respOk(keyFetchTokenResp)
-              const sessionToken = sessionTokenResp
-              const keyFetchToken = keyFetchTokenResp
+              const sessionToken = sessionTokenResp.obj
```

Comment:
Unrelated, but noticed that this wasn't testing against the correct response object.